### PR TITLE
Change PenaltyUs

### PR DESCRIPTION
**Pull request by mamiksik**

### CODE REVIEW

File: src/stp/plays/referee_specific/PenaltyUs.cpp [hunk 1]
```diff
@@ -4,69 +4,84 @@
...
-                                                                                       std::make_unique<role::Halt>(role::Halt("halt_2")),
-                                                                                       std::make_unique<role::Halt>(role::Halt("halt_3")),
-                                                                                       std::make_unique<role::Halt>(role::Halt("halt_4")),
-                                                                                       std::make_unique<role::Halt>(role::Halt("halt_5")),
-                                                                                       std::make_unique<role::Halt>(role::Halt("halt_6")),
-                                                                                       std::make_unique<role::Halt>(role::Halt("halt_7")),
-                                                                                       std::make_unique<role::Halt>(role::Halt("halt_8"))};
+    /// Evaluations that have to be true to be considered when changing plays.
+    startPlayEvaluation.clear(); // DONT TOUCH.
+    startPlayEvaluation.emplace_back(eval::NormalPlayGameState);
```
Comment (JaroKuiken) [inline src/stp/plays/referee_specific/PenaltyUs.cpp]: Should be `startPlayEvaluation.emplace_back(eval::PenaltyUsGameState);`

File: src/stp/plays/referee_specific/PenaltyUs.cpp [hunk 2]
```diff
@@ -4,69 +4,84 @@
...
+
+    /// Evaluations that have to be true to allow the play to continue, otherwise the play will change. Plays can also end using the shouldEndPlay().
+    keepPlayEvaluation.clear(); // DONT TOUCH.
+    keepPlayEvaluation.emplace_back(eval::NormalPlayGameState);
```
Comment (JaroKuiken) [inline src/stp/plays/referee_specific/PenaltyUs.cpp]: The same thing: `keepPlayEvaluation.emplace_back(eval::PenaltyUsGameState);`

**Review by JaroKuiken — CHANGES_REQUESTED** [review]
Make sure to use the right GameState as Evaluation for the Play

File: src/stp/plays/referee_specific/PenaltyUs.cpp (hunk 2 above)
Comment (mamiksik) [reply src/stp/plays/referee_specific/PenaltyUs.cpp]: fixed

File: src/stp/plays/referee_specific/PenaltyUs.cpp (hunk 1 above)
Comment (mamiksik) [reply src/stp/plays/referee_specific/PenaltyUs.cpp]: fixed

**Review by JaroKuiken — APPROVED** [review]

**Pull request merged**